frontend/packages/kubevirt-plugin/integration-tests/tests/vm.tab.console.scenario.ts

```diff
@@ -107,4 +104,34 @@ describe('KubeVirt VM console - VNC/Serial', () => {
 
     expect(extractResult(out)).toEqual('SUCCESS');
   });
+
+  it('ID(CNV-4660) Open VNC Console in new window', async () => {
+    await click(openInNewWindow);
+
+    const consoleWinHandles = await browser.getAllWindowHandles();
+    const parentHandle = consoleWinHandles[0];
+    const popUpHandle = consoleWinHandles[1];
+
+    await browser.switchTo().window(popUpHandle);
+    const getPopUpHandle = browser.getWindowHandle();
+    expect(getPopUpHandle).toEqual(popUpHandle);
+
+    // verify it's connecting to vnc console in the new window.
+    await browser.wait(until.invisibilityOf(vncConnectingBar));
+    await browser.wait(until.presenceOf(vncSendKeyButton), PAGE_LOAD_TIMEOUT_SECS);
+
+    // verify it's connecting to serial console in the new window.
+    await selectDropdownOption(consoleTypeSelectorId, 'Serial Console');
+    await browser.wait(waitForStringNotInElement(serialConsoleWrapper, 'Loading'));
+    await browser.wait(until.presenceOf(serialReconnectButton), PAGE_LOAD_TIMEOUT_SECS);
+    await browser.wait(until.presenceOf(serialDisconnectButton), PAGE_LOAD_TIMEOUT_SECS);
+    await browser.wait(until.elementToBeClickable(serialDisconnectButton), PAGE_LOAD_TIMEOUT_SECS);
+    await browser.wait(until.presenceOf(serialConsole), PAGE_LOAD_TIMEOUT_SECS);
+
```

Comment:
can we also log in into the VM like we did in the other tests? 
I know it shoud work, but there is a different stucture for loading resources, which can break and it would be nice to test it.

Reply:
Done.